README.md

```diff
@@ -8,7 +8,8 @@ The `smartling` CLI tool provides a familiar unix-like command interface to the
 
 ### Installing
  - download from the releases page
- - `go get github.com/99designs/smartling`
+ - `cd <smartling-path>`
```

Comment:
I think the point form is misleading now. Maybe we should break them up and even reduce the methods.
Currently, there are three methods:
1. Download from github release (<== we can remove because github no longer support it for new releases)
2. Install from go source
3. Run from docker image

Reply:
> Download from github release (<== we can remove because github no longer support it for new releases)
Sorry this is actually false. We are still using the github release binary.